voluncheer/.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 100
+max-line-length = 120
```

Comment:
why are you changing the max-line-length for flake8?

Reply:
I ran into an error with flake8 because one of the lines had characters above 100.

Reply:
The line couldn't be broken up?